LibraryApp/src/app/screens/BookDetail/index.js

```diff
@@ -10,12 +15,58 @@ class BookDetail extends Component {
   };
 
   render() {
+    const { item } = this.props.navigation.state.params;
+    const { image_url: coverImage, title, author, year, genre } = item;
     return (
-      <View>
-        <Text>{/* TODO: to give a proper detail to each book */}</Text>
+      <View style={styles.container}>
+        <View style={styles.detailsContainer}>
+          <Image
+            source={(coverImage && { uri: coverImage }) || defaultPlaceholder}
+            style={styles.coverImage}
+          />
+          <ScrollView style={styles.rightContainer}>
+            <Text style={styles.title}>{title}</Text>
+            {/* TODO: Make availability updatable */}
```

Comment:
Do you need this to pass this card?

Reply:
i don't need it because it was not a part of the books.json and i have no api (yet)